src/table/table.module.ts

```diff
@@ -1,47 +1,60 @@
+// modules
 import { NgModule } from "@angular/core";
 import { CommonModule } from "@angular/common";
 import { FormsModule } from "@angular/forms";
+
 import { ChevronRight16Module } from "@carbon/icons-angular/lib/chevron--right/16";
 import { Search16Module } from "@carbon/icons-angular/lib/search/16";
 import { Close16Module } from "@carbon/icons-angular/lib/close/16";
 
-import { DialogModule } from "./../dialog/dialog.module";
 import { NFormsModule } from "./../forms/forms.module";
+import { DialogModule } from "./../dialog/dialog.module";
+import { I18nModule } from "./../i18n/i18n.module";
+
 import { ButtonModule, SearchModule } from "../";
 
+// imports
```

Comment:
```suggestion
// components
```
?

Since `// modules` are also imports